test/blackbox-tests/test-cases/instrumentation/ppx/hello_ppx.ml

```diff
@@ -0,0 +1,16 @@
+open Ast_mapper
+open Ast_helper
+open Longident
+
+let mapper _ _ =
+  { default_mapper with
+    structure = fun _ str ->
+      Str.eval
+        (Exp.apply (Exp.ident (Location.mknoloc (Ldot (Lident "Hello", "hello"))))
+           [Nolabel, Exp.construct (Location.mknoloc (Lident "()")) None]) :: str
+  }
+
+open Migrate_parsetree
+
+let () =
+  Driver.register ~name:"hello" Versions.ocaml_current mapper
```

Comment:
The driver is going to be removed in omp 2.0, so we should add an uperrbound or just use ppxlib right away here.

Reply:
OK, I will try rewriting it with `ppxlib`.

Reply:
This is now fixed (switched to ppxlib).